agent-packaging/packaging-scripts/common.sh

```diff
@@ -21,11 +23,13 @@ export GOPATH=/usr/share/gocode
 export GOCACHE=/tmp/.cache
 
 working_dir=${PWD}
-if [[ $(basename "$working_dir") != $NAME ]]; then
+if [[ $(basename "$working_dir") != "osconfig" ]]; then
```

Comment:
you can delete this check..it was specific to the c-i-p repo to ensure you ran it from the right location for rel paths to work.

Reply:
yeah... i thought i will delete it. but then i realized that to someone who is pretty new to this package build, this check actually provides useful information from where to kick off an agent build. This is only for manual testing of builds (I myself fell into this, i did not know from where to invoke the build scripts. this was specially because the subsequent scripts use the relative path from ${PWD}).